ckanext/security/validators.py

```diff
@@ -58,7 +59,10 @@ def user_about_validator(key, data, errors, context):
 
 invalid_list = ['hacked', 'hack[^a-zA-Z]+by']
 def is_input_valid(input_value):
+    pf = ProfanityFilter()
     for invalid_string in invalid_list:
         if re.search(invalid_string, input_value, re.IGNORECASE):
             return False
+        if not pf.is_clean(input_value):
```

Comment:
Due to the indentation, if the input_value contains any profane words, which does not contain 'hacked', it will not return `False`.